packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -149,34 +141,6 @@ export default defineComponent({
     ...mapGetters('Files', ['currentFileOutgoingCollaborators', 'highlightedFile']),
     ...mapGetters(['configuration', 'user', 'capabilities']),
 
-    helpersEnabled() {
-      return this.configuration?.options?.contextHelpers
-    },
-
-    inviteCollaboratorHelp() {
-      const cernFeatures = !!this.configuration?.options?.cernFeatures
-      return cernFeatures
-        ? {
-            text: shareInviteCollaboratorHelp.text,
-            list: [...shareInviteCollaboratorHelp.list, ...shareInviteCollaboratorHelpCern.list]
-          }
-        : shareInviteCollaboratorHelp
-    },
-
-    spaceAddMemberHelp() {
-      return shareSpaceAddMemberHelp
-    },
-
-    inviteDescriptionMessage() {
-      if (this.capabilities.files_sharing.federation?.outgoing === true) {
-        return this.$gettext('Add new person by name, email or federation IDs')
-      }
-      const cernFeatures = !!this.configuration?.options?.cernFeatures
-      return cernFeatures
```

Comment:
@elizavetaRa @diocas decision was made to remove the description message below the sharing invite form. If you can't drop the line from your UI we'll have to find a different solution. Please reach out then.